test/unit/dimensions.js

```diff
@@ -235,6 +236,45 @@ QUnit.test( "outerWidth()", function( assert ) {
 	div.remove();
 } );
 
+QUnit.test( "outerHeight()", function( assert ) {
```

Comment:
Oh, we *did* but it was further down the file and I missed it. I can remove this and/or beef up the tests there.